src/docs/redis/sections/10-try/30-test.mdx

```diff
@@ -0,0 +1,3 @@
+Send a `PING` command to Redis to test your TLS configuration:
+
+<MDXBlock path="snippets/test" />
```

Comment:
You can also just embed this content directly. I ended up putting it in a snippet for nginx because I wanted to have them test their TLS configuration again after they'd gotten it running in k8s. Then again, maybe we'll want a test step like that after every deplyoment? Should I just duplicate the "test" section down in Operationalize too, at the end?

Reply:
hmm I prefer embedding it directly